include/maths/CMultivariateMultimodalPrior.h

```diff
@@ -386,14 +386,21 @@ class CMultivariateMultimodalPrior : public CMultivariatePrior {
         // where w(i) is its weight we can achieve this by multiplying
         // all weights by some factor f in the range [0, 1].
 
-        if (!this->isForForecasting()) {
```

Comment:
There is nothing in the PR description about removing the "is for forecasting" functionality.  I think it would be good to mention it in the description and explain why it was done because one day somebody might want to search for when and why this functionality was removed.

Reply:
This is dead functionality. The code to set this flag to true is now never called, so this condition is always true.  I spotted it because it is different to the univariate case: it was removed from there but missed from here. I'll add a comment in PR description.